test/BF16/Integration/lit.local.cfg

```diff
@@ -20,5 +20,5 @@ def has_support(feature):
     return True
 
 # AVX512 and SVE should have BF16 support
-if not has_support('avx512') and not has_support('sve'):
+if not has_support('avx512') and not has_support('avx2') and not has_support('sve'):
```

Comment:
SVE has the same support as AVX512. in libxsmm.. not sure why it's excluded

Reply:
sse and avx should be excluded

Reply:
It's not excluded. This is a double negative. It is "supported" is there's AVX2, AVX512 or SVE.

Reply:
> sse and avx should be excluded

It is. These tests are **only** run on AVX2, AVX512 and SVE.